### ISSUE TITLE
Complete list of class labels for the pre-trained NER models?

### ISSUE THREAD

**Issue by fzr72725** [JohnSnowLabs/spark-nlp#312]
## Description

No issues running the model. But I would like to get a complete list of NER labels the pre-trained models provide. I could not find this information anywhere in the existing documentation.

Thanks!

**Comment by saifjsl** [comment]
Thanks, we should update this in the documentation indeed.

For practical purposes, the pretrained NER model has

B-ORG
I-ORG
B-PER
I-PER
B-LOC
I-LOC

and it has been trained from:
https://raw.githubusercontent.com/patverga/torch-ner-nlp-from-scratch/master/data/conll2003/eng.train

**Comment by fzr72725** [comment]
Awesome! Thanks @saifjsl! How can one contribute to the SparkNLP documentation? I can add this into the documentation and would love to join the on-going doc maintenance/update!

**Comment by maziyarpanahi** [comment]
Hi @fzr72725 and thank you for your interest in contributing to this library. For contributing to documentation, you simply need to clone the library and look for `docs` directory in the root. They are bunch of `.html` files like these: 
https://github.com/JohnSnowLabs/spark-nlp/tree/master/docs

To test, you can just use your browser(any) without a need to run anything before hand. (simply double click on the `.html` file and it shows you the exact documentation you see on the Website)

I will also quote the contribution's statement:

> Feel free to clone the repository locally and submit pull requests so we can review them and work together.
> feedback, ideas and bug reports
> testing and development
> training and testing nlp corpora
> documentation and research
https://nlp.johnsnowlabs.com/contribute.html

Thanks again 👍
